### PR TITLE
Hook Display Project Template

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:stop

**Comment by lawrence-lee** [comment]
ci:stop

**Comment by gamerson** [comment]
can you resend these commits but lets rename the project to

`project-templates-war-hook`

File: modules/sdk/project-templates-hook-display/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>com.liferay.portal</groupId>
+			<artifactId>com.liferay.portal.kernel</artifactId>
+			<version>2.0.0</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
```
Comment (gamerson) [inline modules/sdk/project-templates-hook-display/src/main/resources/archetype-resources/pom.xml:23]: Don't need portlet and servlet apis in a hook.

File: modules/sdk/project-templates-hook-display/src/main/resources/archetype-resources/src/main/java/__className__.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+/**
+ * @author ${author}
+ */
+public class ${className} extends SimpleAction {
+	public ${className}() {
+		super();
+	}
+
+	public void run(String[] arg0) throws ActionException {
```
Comment (gamerson) [inline modules/sdk/project-templates-hook-display/src/main/resources/archetype-resources/src/main/java/__className__.java:14]: This should print out so and so logged in

**Pull request closed without merging**